torch/_dynamo/variables/builtin.py

```diff
@@ -1299,6 +1300,16 @@ def _unimplemented():
         if isinstance(left, ConstantVariable) and isinstance(right, ConstantVariable):
             return ConstantVariable(op(left.value, right.value))
 
+        if isinstance(left, UserDefinedObjectVariable) and isinstance(
+            right, UserDefinedObjectVariable
+        ):
+            return ConstantVariable(op(left.value, right.value))
+
+        if op.__name__ == "is_":
```

Comment:
Unforunately, this is wrong. `SymNodeVariable(2) is ConstantVariable(2)` but their VariableTracker types are different.